daemon/container.go

```diff
@@ -811,7 +811,7 @@ func (container *Container) exec(ExecConfig *ExecConfig) error {
 	container.Lock()
 	defer container.Unlock()
 
-	callback := func(processConfig *execdriver.ProcessConfig, pid int) {
+	callback := func(processConfig *execdriver.ProcessConfig, pid int, ch <-chan struct{}) {
```

Comment:
Here and below, can you change the parameter name to something like `oomCh`, just `ch` sounds too generic